src/extension.ts

```diff
@@ -29,23 +30,25 @@ export async function activate(context: ExtensionContext): Promise<void> {
     workspace.registerTextDocumentContentProvider(PrettyAnsiContentProvider.scheme, prettyAnsiContentProvider)
   );
 
-  const showPretty = async (options?: TextDocumentShowOptions) => {
-    const actualUri = window.activeTextEditor?.document.uri;
+  const showPretty = async (forcePreview: boolean = false, options: TextDocumentShowOptions = { viewColumn: ViewColumn.Active}) => {
+    if (forcePreview || workspace.getConfiguration(`${extensionId}`).autoPreview) {
+      const editor = window.activeTextEditor
+      if (editor && editor.document && editor.document.languageId === 'ansi' && editor.document.uri.scheme === 'file') {
+        const providerUri = PrettyAnsiContentProvider.toProviderUri(editor.document.uri);
 
-    if (!actualUri) {
-      return;
-    }
-
-    const providerUri = PrettyAnsiContentProvider.toProviderUri(actualUri);
+        const autoPreviewToSide = workspace.getConfiguration(`${extensionId}`).autoPreviewToSide;
```

Comment:
```suggestion
        const autoPreviewToSide = workspace.getConfiguration(extensionId).autoPreviewToSide;
```

This template literal is unnecessary.